cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -43,12 +42,58 @@ import           Data.Yaml.Pretty (setConfCompare)
 import qualified Data.Yaml.Pretty as Yaml
 import           GHC.Real (denominator)
 import           GHC.Unicode (isAlphaNum)
+import qualified Data.Aeson.Encode.Pretty as Aeson
+import Control.Monad.Trans (MonadIO)
+
+jsonConfig :: Aeson.Config
+jsonConfig = Aeson.defConfig {Aeson.confCompare = compare}
+
+friendlyTxJson ::
+  (MonadIO m, IsCardanoEra era)
```

Comment:
```suggestion
  (MonadIO m)
```
Redundant constraint. `IsCardanoEra era` is obtainable using `cardanoEraConstraints` function.